src/index.js

```diff
@@ -226,7 +232,7 @@ class DropdownTreeSelect extends Component {
                   onAction={this.onAction}
                   onCheckboxChange={this.onCheckboxChange}
                   onNodeToggle={this.onNodeToggle}
-                  simpleSelect={this.props.simpleSelect}
+                  simpleSelect={isSimpleSelect}
```

Comment:
I feel that this could lead to potential issues down the line as it alters the meaning of `simpleSelect`. Up until now, `simpleSelect` lets you _safely_ assume certain things like no need to traverse below level 1, and the drop-down is a basically flat list. With this change, those assumptions are no longer valid and so it could become a potential source of trouble.

Thoughts?

Reply:
This is like a hybrid which behaves like a simpleSelect but allows the traversing to be done. If you want it separated all the way we need to add checks and props everywhere where simpleSelect is used. The current solution allows it to behave as a simpleSelect everywhere except in the treemanager where it makes a difference.

Reply:
> The current solution allows it to behave as a simpleSelect everywhere except in the treemanager where it makes a difference.

For now, yes. And I see how this could have been misleading.

But let me look into this locally and I'll let you know.

Reply:
I looked into this a bit and I think we should separate it out. While separating it out results in more work on us, I think the benefits are worth the effort. E.g., right now, with `simpleSelect`, information is _lost_ in a sense that I can't tell whether it's simple or single. Secondly, we're adding CSS classes for easy targeting `className={cx('checkbox-item', { 'simple-select': simpleSelect })}` and if one wants to target simple differently than single, it is impossible to do so now.

Further, as I was looking into this, I realized, why are we making checkboxes act like radio buttons? Isn't rendering radio instead of checkbox semantically more correct? 

And what do you think about choosing a different name because `single` sounds too similar to `simple`. Maybe `radioSelect` or something else?

Reply:
I need the id set from the other PR to set a shared name (a.k.a the dropdown id) for the radio buttons.